wire-swift-generator/src/main/java/com/squareup/wire/swift/SwiftGenerator.kt

```diff
@@ -930,9 +935,21 @@ class SwiftGenerator private constructor(
     includeDefaults: Boolean = true
   ) = apply {
     type.fields.forEach { field ->
-      val fieldType = field.typeName
+      var typeName = when (field.type) {
+        ProtoType.BYTES -> if (field.isMap) {
+          dataMap
```

Comment:
This seems wrong?
I would definitely want test cases for things like `[String: Data]`, `[Data: String]` (if that's possible), etc

Reply:
makes sense

Reply:
done should be fixed now.